core/schemes/base.php

```diff
@@ -21,6 +21,8 @@ abstract class Base {
 	 */
 	const LAST_UPDATED_META = '_elementor_scheme_last_updated';
 
+	const SCHEME_OPTION_PREFIX = 'elementor_scheme_';
```

Comment:
Revert changes for this file because you delete the option on upgrade

Reply:
Let talk about this in private, I think I didn't understand the task correctly.